app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,85 @@
+class InventoriesController < ApplicationController
+    def index
+      @inventories = Inventory.all
+    end
+  
+    def new
+      @inventory = Inventory.new
+    end
+  
+    def create
+      @inventory = Inventory.new(inventory_params)
+      if @inventory.save
+        redirect_to @inventory, notice: 'Inventory item was successfully created.'
+      else
+        render :new
+      end
+    end
+  
+    def show
+      @inventory = Inventory.find(params[:id])
+    end
+  
+    def edit
+      @inventory = Inventory.find(params[:id])
+    end
+  
+    def update
+      debugger
+      @inventory = Inventory.find(params[:inventory_id])
```

Comment:
What if @inverntory is nil? and params variable doesn't have any inventory_id. S

Please make you code resilient so that it can handle all negative cases and security cases as well.